tool/live-monitor.js

```diff
@@ -0,0 +1,166 @@
+const Web3 = require('web3');
+const { getBridgeTransactionByTxHash } = require('../index');
+const networkParser = require('./network-parser');
+const Bridge = require('@rsksmart/rsk-precompiled-abis').bridge;
+const util = require('util');
+
+const DEFAULT_CHECK_EVERY_MILLIS = 1_000;
+
+const defaultCommandsValues = {
+    fromBlock: 'latest',
+    methods: [],
+    events: [],
+    pegout: false,
+    pegin: false,
+    network: 'testnet',
+    checkEveryMillis: DEFAULT_CHECK_EVERY_MILLIS,
+};
+
+const getParsedCommands = () => {
+    const commands = process.argv.filter(command => command.startsWith('--'))
+    .reduce((commands, command) => {
+        if(command.startsWith('--fromblock')) {
+            commands.fromBlock = command.slice(command.indexOf('=') + 1);
+        } else if(command.startsWith('--methods') || command.startsWith('--events')) { // Parsing commands that include an array
+            commands['methods'] = JSON.parse(command.slice(command.indexOf('=') + 1).replaceAll("'", '"'));
+        } else if(command.startsWith('--network')) {
+            commands.network = command.slice(command.indexOf('=') + 1);
+        } else if(command.startsWith('--pegout') || command.startsWith('--pegin')) {
+            const parsedCommand = command.replace('--', '');
+            commands[parsedCommand] = true;
```

Comment:
Maybe we can do this same thing with all the commands? To avoid having some many else if

Reply:
I don't think I understand what you mean here. Some of the parameters are different and need to be parsed differently. The ones that are similar are handled in the same if block.